sdk/helper/ldaputil/connection.go

```diff
@@ -20,3 +20,8 @@ type Connection interface {
 	SetTimeout(timeout time.Duration)
 	UnauthenticatedBind(username string) error
 }
+
+type ConnectionPaging interface {
```

Comment:
Nit: The other way around (`PagingConnection`) makes more sense as an interface name imo.

Reply:
I like that change, updated the name